onnxruntime/core/util/protobuf_parsing_utils.h

```diff
@@ -3,284 +3,6 @@
 
 #include <string>
```

Comment:
Do we still need `<string> and `<iosfwd>`?

Reply:
Thank you! I removed them.